scripts/qa_crowbarsetup.sh

```diff
@@ -1053,6 +1054,60 @@ EOF
         /opt/dell/bin/json-edit -a attributes.network.conduit_map -r -v "$conmap_complete" $netfile
     fi
 
+    if [[ $want_separate_storage_network = 1 ]]; then
+        # special conduit for a separate storage network
+
+        # append conduit intf3 to team conduit_maps
+        # (maps must contain the same list of conduits as the catch-all list)
+        temp_py=`mktemp`
+        cat << EOF > $temp_py
+import json
+
+team1_conlist_element = {
+    'intf3': {
+        'if_list': [
+            '?1g1'
+        ]
+    }
+}
+team_conlist_element = {
+    'intf3': {
+        'if_list': [
+            '1g3',
+            '1g4'
+        ]
+    }
+}
+
+with open("$netfile") as f:
+    j = json.load(f)
+
+dirty1 = False
+dirty = False
+for conmap_el in j['attributes']['network']['conduit_map']:
+    if conmap_el['pattern'] == 'team/1/crowbar':
+        conmap_el['conduit_list'].update(team1_conlist_element)
+        dirty1 = True
+    elif conmap_el['pattern'] == 'team/.*/.*':
+        conmap_el['conduit_list'].update(team_conlist_element)
+        dirty = True
+if dirty1 and dirty:
+    print json.dumps(j['attributes']['network']['conduit_map'], indent=4)
+fi
+EOF
+        local conmap_complete=`python $temp_py`
+        rm -f $temp_py
+        if [ -z "$conmap_complete" ]; then
+            complain 1024 "Failed to append intf3"
+        fi
+        # apply the modified conduit map
+        /opt/dell/bin/json-edit -a attributes.network.conduit_map -r -v "$conmap_complete" $netfile
+        # set storage network conduit to intf3
+        /opt/dell/bin/json-edit -a attributes.network.networks.storage.conduit -r -v "\"intf3\"" $netfile
+        # disable storage network vlan
+        /opt/dell/bin/json-edit -a attributes.network.networks.storage.use_vlan -r -v "false" $netfile
```

Comment:
I am not sure we wanted to disable vlan there. How would the VM interfaces be connected? I believe they are all connected to the same cloudbridge, where untagged traffic is already reserved for the admin network.